shuup/core/models/_products.py

```diff
@@ -494,7 +494,11 @@ def get_price(self, context, quantity=1):
         :type context: shuup.core.pricing.PricingContextable
         :rtype: shuup.core.pricing.Price
         """
-        return self.get_price_info(context, quantity).price
+        # Check if product has variations (children) if yes, return cheapest variation price
+        if not self.get_priced_children(context, quantity):
```

Comment:
I guess this is a super expensive check.. IMO check the product mode instead. If it is a parent, then return the cheapest child. Thoughts @tulimaki ?